.test-defs/SeedLoggingTest.yaml

```diff
@@ -0,0 +1,21 @@
+kind: TestDefinition
+metadata:
+  name: seedlogging-test
+spec:
+  owner: i.alidjikov@gmail.com
+  recipientsOnFailure: i.alidjikov@gmail.com
```

Comment:
We changed this field to a list of emails as discussed yesterday.
Can you please update this to the new structure.
```yaml
recipientsOnFailure:
- i.alidjikov@gmail.com
```

Reply:
Thank you for your review. Updated the PR.